src/components/PhoneTextInput/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import * as baseTextInputPropTypes from '../TextInput/baseTextInputPropTypes';
+import TextInput from '../TextInput';
+import LoginUtil from '../../libs/LoginUtil';
+
+class PhoneTextInput extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            val: props.value || props.defaultValue || '',
+        };
+        this.setValue = this.setValue.bind(this);
+    }
+
+    setValue(val) {
+        this.setState({val});
+        let phoneNumber = LoginUtil.getPhoneNumberWithoutSpecialChars(val);
+
+        if (this.props.VBAPhone) {
+            if (/^\+1/.test(phoneNumber)) {
+                phoneNumber = phoneNumber.replace(/^\+?1|\|1|\D/, '');
+            }
+        }
```

Comment:
What does this change do? I don't think this is needed

Reply:
@parasharrajat, this is to remove +1 from the phoneNumber in CompanyStep and AdditionalDetailsStep

Reply:
Ok then please use the general method to get the phoneNumber without countryCode and remove the extra regex. There is one can't remember the name.

Reply:
@parasharrajat, are you referring to this [function ](https://github.com/Expensify/App/blob/6ac9fc098d46a16cfb3318cd652302a34a7fa2a6/src/libs/LocalePhoneNumber.js#L15-L15)
I think we can use this by passing the local as en and the number directly(as it doesn't remove +1 for spanish local)